deploy/gk-deploy

```diff
@@ -239,7 +239,7 @@ while [[ $# -ge 1 ]]; do
   shift
 done
 
-if [[ ! -f ${TOPOLOGY} ]]; then
+if [[ ${ABORT} -ne 1 ]] && [[ ! -f ${TOPOLOGY} ]]; then
```

Comment:
Okay, fair. However, we still have to worry about the solution to #81 because as it stands it introduces a need for the topology file in `abort()` thus these two PRs conflict with each other.